src/app/material-docs-app.scss

```diff
@@ -26,6 +26,28 @@ material-docs-app > guide-viewer {
   overflow-y: auto;
 }
 
+#skip-link-wrapper {
+  z-index: 10;
+  position: fixed;
+  top: 15px;
+  width: 16%;
+  margin: 0 42% 0 42%;
+  text-align: center;
+  padding: 0.2em 0 0.2em 0;
+  background: white;
+  border-radius: 20px;
+  opacity: 0;
```

Comment:
Is `opacity: 0` sufficient to prevent the element from receiving click events?

Reply:
Yeah just checked this, `opacity:0` is not sufficient. We have a class in the CDK called `cdk-visually-hidden` that should handle this, I would just apply that class to the element